sc/social/like/configure.zcml

```diff
@@ -34,6 +34,34 @@
       handler=".subscribers.assign_canonical_url"
       />
 
+  <subscriber
+      zcml:condition="installed plone.app.dexterity"
+      for="plone.dexterity.interfaces.IDexterityContent
+           Products.DCWorkflow.interfaces.IAfterTransitionEvent"
+      handler=".subscribers.social_content_check_workflow"
+      />
+
+  <subscriber
+      zcml:condition="installed plone.app.dexterity"
+      for="plone.dexterity.interfaces.IDexterityContent
+           plone.dexterity.interfaces.IEditFinishedEvent"
+      handler=".subscribers.social_content_check"
+      />
+
+  <subscriber
```

Comment:
to simplify development this new feature can be implemented for Dexterity-based content types only.

Reply:
ok